packages/components/src/components/code-snippet/_code-snippet.scss

```
@@ -199,12 +199,13 @@
     right: 1.25rem;
   }
 
-  .#{$prefix}--btn--copy__feedback:before,
-  .#{$prefix}--btn--copy__feedback:after {
+  .#{$prefix}--btn--copy__feedback::before,
+  .#{$prefix}--btn--copy__feedback::after {
+    @include type-style('code-01');
```

```suggestion
    @include type-style('body-short-01');
```

Would this be the right change based on @laurenmrice's feedback?

---

yes unless the feedback message is also supposed to be `code-01` cc @laurenmrice

---

The feedback message is inside of a tooltip, so it should be getting `body-short-01` not `code-01`

---

@joshblack just including that mixin won't override the font because we don't include the `font-family` rule in `type-style` (https://github.com/carbon-design-system/carbon/issues/4551)

should we just manually set the font here or do we want to extend https://github.com/carbon-design-system/carbon/pull/4440?